frontend/packages/dev-console/src/components/topology/index.ts

```diff
@@ -5,3 +5,14 @@ export {
 } from './relationship-provider';
 export * from './hpa-tab-section';
 export * from './observe-tab-section';
+export { getBindableServiceResources } from './bindable-services/data-transformer';
+
+export const getBindableServicesTopologyDataModel = () =>
+  import(
+    './bindable-services/data-transformer' /* webpackChunkName: "topology-bindable-services" */
+  ).then((m) => m.getBindableServicesTopologyDataModel);
+
+export const isServiceBindable = () =>
+  import(
+    './bindable-services/isBindable' /* webpackChunkName: "topology-bindable-services" */
+  ).then((m) => m.isServiceBindable);
```

Comment:
This is going to pull a lot of unnecessary dependencies into your main and vendor bundles. Biggest one being `@patternfly/react-topology` which will blow up the analyze CI. Keep these into there own files maybe and import them into `plugin.tsx` directly.

Reply:
Moved `getBindableServicesTopologyDataModel` to it's own file, and imported directly into plugin file.